docs/network_firewall.rst

```diff
@@ -99,6 +99,38 @@ chosen:
 -  Monitor Gateway: ``10.20.3.1``
 -  Monitor Server (OPT2) : ``10.20.3.2``
 
+3 NIC Example (SG-3100)
```

Comment:
It it possible to make the numbering 5. b. for `3 NIC Example` and 5. a. for the `4 NIC Example`? Admins might think they must complete both.

Reply:
I attempted to do here but it actually looks much worse and is less clear. I did make that change in the other suggested section.